### DISCUSSION TITLE
Welcome to awesome-table-structure-recognition Discussions!

### DISCUSSION THREAD

**Discussion by Tan-Junwen** [MathamPollard/awesome-table-structure-recognition#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Arlen-yuzu** [comment]
您好，请问关于SEMv2这篇工作的开源代码，您有找到数据预处理的方案吗？原作者代码里面的模型输入的训练数据里，'row_start_center_bboxes' 和 'row_line_segmentations' 的属性似乎都找不到怎么得到的。
